cdc/sink/table_sink.go

```diff
@@ -88,7 +88,12 @@ func (t *tableSink) flushResolvedTs(ctx context.Context, resolvedTs uint64) (uin
 	if redoTs < resolvedTs {
 		resolvedTs = redoTs
 	}
-	return t.manager.flushBackendSink(ctx, t.tableID, resolvedTs)
+
```

Comment:
There is a function adjustment involved here.

Reply:
Is there any particular reason to modify it? Does it feel like the original is better encapsulated?

Reply:
There is no special reason, just to make it easier to read the code.